ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt

```diff
@@ -8,26 +8,69 @@ org.jboss.spec.javax.jms:jboss-jms-api_2.0_spec
 org.jboss.spec.javax.transaction:jboss-transaction-api_1.3_spec
 
 com.google.guava:guava
+com.google.guava:failureaccess
 commons-beanutils:commons-beanutils
 commons-collections:commons-collections
-io.netty:netty-all
+io.netty:netty-buffer
+io.netty:netty-resolver
+io.netty:netty-transport
+io.netty:netty-handler
+io.netty:netty-handler-proxy
+io.netty:netty-codec
+io.netty:netty-codec-socks
+io.netty:netty-common
+io.netty:netty-transport-native-unix-common
+io.netty:netty-transport-native-unix-common:linux-x86_64
+io.netty:netty-transport-native-unix-common:linux-aarch_64
+io.netty:netty-transport-classes-epoll
+io.netty:netty-transport-native-epoll:linux-x86_64
+io.netty:netty-transport-native-epoll:linux-aarch_64
+io.netty:netty-transport-classes-kqueue
+io.netty:netty-codec-http
+io.undertow:undertow-core
+jakarta.jms:jakarta.jms-api
+jakarta.transaction:jakarta.transaction-api
+jakarta.ejb:jakarta.ejb-api
+jakarta.resource:jakarta.resource-api
```

Comment:
the above 4 jakarta api should move to the top to replace the no-longer-used javax api artifacts.

Reply:
Made the required changes.